operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -514,6 +515,71 @@ function createModelDomain() {
   trace "Exiting createModelDomain"
 }
 
+
+# Expands into the root directory the MII domain configuration, stored in one or more config maps
+function restoreDomainConfig() {
+  restoreEncodedTar "domainzip.secure" || return 1
+
+  chmod +x ${DOMAIN_HOME}/bin/*.sh ${DOMAIN_HOME}/*.sh  || return 1
+}
+
+# Expands into the root directory the MII primordial domain, stored in one or more config maps
+function restorePrimordialDomain() {
+  restoreEncodedTar "primordial_domainzip.secure" || return 1
+}
+
+# Restores the specified directory, targz'ed and stored in one or more config maps after base 64 encoding
+# args:
+# $1 the name of the encoded file in the config map
+function restoreEncodedTar() {
+  cd / || return 1
+  indexRange="$(getIndexRange $1)"
+  # shellcheck disable=SC2046
+  cat $(buildConfigMapElements $1 $indexRange) > /tmp/domain.secure
```

Comment:
Alternatively: `cat ${OPERATOR_ROOT}/introspector*/${1} > /tmp/domain.secure`.

Reply:
Thanks. That both simplifies the code and eliminates the need for the additional functions and their tests.